db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/river/LagreJournalpostIdLoeser.kt

```diff
@@ -72,7 +72,7 @@ class LagreJournalpostIdLoeser(
         if (journalpostId.isNullOrBlank()) {
             logger.error("Fant ingen journalpost-ID.")
             sikkerLogger.error("Fant ingen journalpost-ID.")
-            behov.createFail("Klarte ikke lagre journalpostId for transaksjonsId $transaksjonId. Tom journalpost-ID!")
```

Comment:
Ikke viktig, men det er vel egentlig mest vanlig å si / bruke "transaksjonsid" med S.

Reply:
Ja, men tror det er grammatisk feil. ID er en forkortelse, og da skal man bruke bindestrek i kombinasjon med andre ord. Og når vi bruker camelcasing så burde ikke "s" være med for det casingen erstatter bindestreken.

(Min unnskyldning for at jeg gidder å bruke tid på dette er at jeg var med i universitetsavisa 🤓)